glossary.md

```diff
@@ -32,6 +32,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 |-------------------------|-------------------------|-----------------------|
 | background noise        | nhiễu nền               | http://bit.ly/31ObyKI |
 | back-propagation        | lan truyền ngược        |                       |
+| backward pass | lượt tính ngược | |
```

Comment:
mình có `backprop` là `lan truyền ngược` rồi,  hay`backward pass` dịch thành `lượt truyền ngược` cho nó giống nhỉ?
Tương tự `forward pass` thành `lượt truyền xuôi`.
VD: "In the forward pass, model will store activations of each layer to backpropagate gradients in the backward pass." -> "Trong lượt truyền xuôi, mô hình sẽ lưu giá trị kích hoạt của mỗi tầng để lan truyền ngược gradient tại lượt truyền ngược".

P/s: Mình chưa ưng chữ `lượt` lắm.

Reply:
Mình thường dịch deploy là hiện thực hóa (tức từ code ra production), implement là triển khai (từ idea/pseudo sang code). Còn từ "xây dựng" dễ bị hiểu nhầm là build ấy.